components/brave_ads/common/pref_names.cc

```diff
@@ -30,6 +30,12 @@ const char kSupportedCountryCodesSchemaVersion[] =
     "brave.brave_ads.supported_regions_schema_version_number";
 const int kSupportedCountryCodesSchemaVersionNumber = 9;
 
+// Stores the last origin of the ad notification
```

Comment:
just curious what are X and Y? Maybe it's worth to expand the comment

Reply:
Changed, thanks